doc/project/project_roles.rst

```diff
@@ -314,15 +314,46 @@ Release Activity
 Merge Criteria
 ++++++++++++++
 
-* All continuous integration checks have passed
+* Minimal of 2 approvals from the list below, including an approval by the
+  designated assignee:
+
+  * Collaborators from the same subsystem.
+  * Maintainers and collaborators of other areas.
+
+* Four eye principle on the organisation level. We already require at least 2
+  approvals (basic four eye principle), however, such reviews and approvals
+  might be unintentionally biased in the case where the submitter is from the
+  same organisation as the approvers. To allow for project wide review and
+  approvals, the merge criteria is extended with the guidelines below:
+
+  * Changes or additions to common and shared code shall have reviews from
+    different organisations (at least one review and approval from an
+    organisation different than the submitters').
+    Common and shared code is defined as anything that does not fall under
+    :file:`soc`, :file:`boards` and :file:`drivers/*/*`.
+  * Changes or additions to hardware support (driver, SoC, boards) shall at
+    least have the merger be from a different organisation. This applies only
+    to implementation of an API supporting vendor specific hardware and not the
+    APIs.
+  * Release engineers shall not merge code changes originating and reviewed
+    only by their own organisation. To be able to merge such changes, at least
```

Comment:
Isn't "To be able to merge such changes, at least one review shall be from a different organisation."
fully redundant with the previous sentence?

Reply:
it is saying it in a different way.